packages/server/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "nodemon",
     "build": "tsc",
-    "migrate": "npx prisma migrate dev",
```

Comment:
Can we use `pnpx` instead. This will guarantee run

Reply:
Anyway we will be `npx` or `pnpx` while uisng the command. So the resultant command becomes `pnpx pnpx prisma migrate dev`. So in the scripts only base command need to be included

Reply:
No ! i don't think NX works in this way  It just finds the target package and run the command provided.

Reply:
All the commands will be executed from the root dir.

Reply:
yes right ut nx will append `pnpx` internally, so it was throwing error if we include `npx` in the script itself

Reply:
Ok